doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_sth_sink.md

```diff
@@ -330,6 +330,7 @@ A configuration example could be:
     cygnus-ngsi.sinks.sth-sink.mongo_hosts = 192.168.80.34:27017
     cygnus-ngsi.sinks.sth-sink.mongo_username = myuser
     cygnus-ngsi.sinks.sth-sink.mongo_password = mypassword
+    cygnus-ngsi.sinks.sth-sink.mongo_auth_source = admin
```

Comment:
The new parameter mongo_auth_source should be also added to the configuration table included in this file .md file.

Reply:
The new parameter mongo_auth_source have been added to the configuration table for both sinks.